CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ Mapbox welcomes participation and contributions from everyone.
 #### Features
 
 #### Bug fixes and improvements
+
+## Mapbox Navigation SDK 2.4.0-rc.1 - March 31, 2022
+
+#### Bug fixes and improvements
+- Optimized the alternative routes references creation since the native peer is now directly delivered from NN. [#5629](https://github.com/mapbox/mapbox-navigation-android/pull/5629)
```

Comment:
We can drop this, too many internal references to be understandable to develoeprs.